src/lib/expression/evaluation/like_matcher.hpp

```diff
@@ -82,19 +82,23 @@ class LikeMatcher {
    *    LikeMatcher{"%hello%"}.resolve(false, [](const auto& matcher) {
    *        std::cout << matcher("He said hello!") << std::endl;
    *    }
+   *
+   * The first template type T defaults to pmr_string, but can also be std::string_view (e.g., for fixed string
+   * dictionary segments). The template type for the functor is the second template argument, since it usually
+   * deduced while T is not decuded automatically.
```

Comment:
typo

Reply:
Comment might be removed when `auto` works as expected. But do you mean the missing "is"? Cannot find a typo.

Reply:
decuded